client/js/components/fragments/Header.js

```diff
@@ -10,24 +10,21 @@ class Header extends Component {
   render() {
     const { user, onLogout, onLogoutSuccess } = this.props;
 
-    var header;
+    let navigation;
     if (user.id) {
-      header = <HeaderAuthenticated user={user} onLogout={onLogout} onLogoutSuccess={onLogoutSuccess}/>;
+      navigation = <HeaderAuthenticated user={user} onLogout={onLogout} />;
```

Comment:
this can be a mistake in the rebasing. I'm not sure why I removed this. probably a mistake, will take a look.